### PR TITLE
Combine Scheme and Multisig

### PR DESCRIPTION
**Pull request by Ghost-Slime**
EXPL-96: Combine Scheme and Multisig

### CODE REVIEW

File: src/modules/account/views/ViewAccount.vue [hunk 1]
```diff
@@ -32,8 +30,7 @@ import AccountComponent from "@/modules/account/components/AccountComponent.vue"
 import AssetComponent from "@/modules/account/components/AssetComponent.vue";
 import NamespaceComponent from "@/modules/account/components/NamespaceComponent.vue";
 import MetadataComponent from "@/modules/account/components/MetadataComponent.vue";
-import MultisigComponent from "@/modules/account/components/MultisigComponent.vue";
-import SchemeComponent from "@/modules/account/components/SchemeComponent.vue";
+import MultisigComponent from "../components/MultisigComponent.vue";
```
Comment (shinneng) [inline src/modules/account/views/ViewAccount.vue:33]: what's the difference between `@/modules/account/components/MultisigComponent.vue` and `../components/MultisigComponent.vue`

File: src/modules/account/components/MultisigComponent.vue [hunk 2]
```diff
@@ -100,20 +117,38 @@ export default {
       return cosignerAccountsList
     }, {deep: true} )
 
+    const strPublicKey = computed(()=>{
+      let strPublicKey = props.PublicKey;
+      return  strPublicKey
+    })
+
```
Comment (anakin001) [inline src/modules/account/components/MultisigComponent.vue:124]: Props are immutable, this function is unnecessary, can just use props.publicKey

File: src/modules/account/components/MultisigComponent.vue [hunk 3]
```diff
@@ -100,20 +117,38 @@ export default {
...
+    const strAddress = computed(()=>{
+      let strAddress = props.address;
+      return strAddress
+    })
+
```
Comment (anakin001) [inline src/modules/account/components/MultisigComponent.vue:129]: Same as above

**Review by anakin001 — CHANGES_REQUESTED** [review]

File: src/modules/account/views/ViewAccount.vue (hunk 1 above)
Comment (Ghost-Slime) [reply src/modules/account/views/ViewAccount.vue:33]: It is basically the same but ../components/MultisigComponent.vue is short form of  @/modules/account/components/MultisigComponent.vue. I will change it.

File: src/modules/account/components/MultisigComponent.vue (hunk 2 above)
Comment (Ghost-Slime) [reply src/modules/account/components/MultisigComponent.vue:124]: Noted

Comment (anakin001) [reply src/modules/account/components/MultisigComponent.vue:123]: i mean you dont need this function at all, you can use strPublicKey on html and props.strPublicKey in script

**Pull request merged**